_visual-editor/README.md

```diff
@@ -0,0 +1,381 @@
+# Migrating from `flutter-quill` to `visual-editor`
+
+> [!WARNING]
+>
+> At the time of writing,
+> `visual-editor` is still in development and **does not offer all the features `flutter-quill` does**.
+> There are a few bugs that are noticeable (more specifically, text selection) 
+> that do not work properly on mobile devices.
+>
+> We will use a **fork of `visual-editor`** - https://github.com/LuchoTurtle/visual-editor in this guide.
+> This is because the PR that was opened to `visual-editor` has need yet been merged
+> (https://github.com/visual-space/visual-editor/pull/237).
```

Comment:
If the PR(s) you open to `visual-editor` are not being merged after _weeks_ of waiting
and they are legitimately valid enhancements then it could be time to:
a) consider other options
b) use the `fork` for the foreseable future and forget the `PRs` which will bit-rot ...